src/euphorie/content/browser/survey.py

```diff
@@ -520,3 +522,51 @@ def tools(self):
                 # we're done if we found at least one measure
                 break
         return tools
+
+
+class ListLinks(BrowserView):
+    attributes_checked = [
+        "description",
+        "introduction",
+        "solution_direction",
+        "legal_reference",
+        "action",
+        "action_plan",
+        "prevention_plan",
+        "requirements",
+    ]
```

Comment:
Are you sure these are all the fields in all relevant content types?
You could dynamically get all text fields and iterate over them.  See how [collective.searchandreplace does it](https://github.com/collective/collective.searchandreplace/blob/master/collective/searchandreplace/searchreplaceutility.py#L247-L249).  Ignore the Archetypes-related code. ;-)